src/fxos/index.js

```diff
@@ -0,0 +1,13 @@
+let modularize = require('../modularize');
+
+let Fxos = modularize(function Fxos(adb) {
```

Comment:
Frankly I don't see the point of the first option, since that is effectively what the `Fxos` middleware is already doing. The second one has value, but at the expense of verbosity with method arguments. Your choice!